README.md

```diff
@@ -130,6 +133,8 @@ class MovieTicketPaid extends Notification
             ->body(Component::dateTime(new \DateTimeImmutable))
             ->body(Component::text('Star Wars'))
             ->body(Component::text('5'))
+            ->buttons(Component::quickReplyButton(['Thanks for your reply!']))
+            ->buttons(Component::urlButton(['reply/01234'])) // List of url suffixes
```

Comment:
Is this always a suffix?

Reply:
According to the Message Template UI (see pic below) only one variable can be added at the end of the website, for dynamic URLs. This only applies to Url Buttons.
<img width="1090" alt="Screenshot 2023-06-05 at 13 40 04" src="https://github.com/netflie/laravel-notification-whatsapp/assets/6515216/a0964cdb-42e9-4b57-987f-b9bffb83ddb9">
[More info](https://business.facebook.com/business/help/2055875911147364?id=2129163877102343)